ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,732 @@
+{
```

Comment:
<p>Should use the same name everywhere:</p><p>Vertex SDK for Python</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>